bundles/net.tourbook/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -125,15 +159,26 @@ sp_cleanup.remove_trailing_whitespaces_ignore_empty=false
 sp_cleanup.remove_unnecessary_array_creation=false
 sp_cleanup.remove_unnecessary_casts=true
 sp_cleanup.remove_unnecessary_nls_tags=true
-sp_cleanup.remove_unused_imports=true
+sp_cleanup.remove_unused_imports=false
```

Comment:
Can you fix this file, this morning I spend time to revert your deletion and now it is modified again. 
This value `sp_cleanup.remove_unused_imports=true` cannot be `false`

All of these files must be the same

![mt ui prefs](https://user-images.githubusercontent.com/1283445/114303870-90507880-9ad0-11eb-93d8-799caeefca2c.png)